tests/test_datasets.py

```diff
@@ -0,0 +1,15 @@
+"""Tests for nlp4all dataset functions."""
+import unittest
+
+import pytest
+
+from nlp4all.datasets import split_dict
+
+class DatasetTest(unittest.TestCase):
```

Comment:
Why this?

Reply:
Oh this is for two reasons:

1. Unit Testing a function that is used in the application; and
2. Providing an example of how to structure the tests for when we want to add tests for other files/classes/functions

This also shows the difference between unit testing and functional / integration testing

Read more about the [difference between unit and functional testing here](https://stackoverflow.com/questions/2741832/what-is-the-difference-between-unit-tests-and-functional-tests#2741845) and the difference between [functional and integration testing here](https://stackoverflow.com/questions/3670729/what-is-the-difference-between-integration-testing-and-functional-testing)

At some point we can verily introduce a headless browser (chrome, Firefox) that will do integration testing to emulate the most important actions in the app - e.g. visit login page, type username and password, click login, upload a file, analyze the file, create NLP pipeline, and then logout